lib/readline.js

```diff
@@ -602,8 +602,8 @@ Interface.prototype._wordRight = function() {
 function charLengthLeft(str, i) {
   if (i <= 0)
     return 0;
-  if (i > 1 && str.codePointAt(i - 2) >= kUTF16SurrogateThreshold ||
-      str.codePointAt(i - 1) >= kUTF16SurrogateThreshold) {
+  if (i > 1 && (str.codePointAt(i - 2) >= kUTF16SurrogateThreshold ||
+      str.codePointAt(i - 1) >= kUTF16SurrogateThreshold)) {
```

Comment:
This seems to be a typo. The condition does not seem to be identical to the original one.

Reply:
Good catch.